test/gutenberg-test-themes/block-bindings/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Block Bindings
+Theme URI: https://github.com/wordpress/gutenberg/
+Author: the WordPress team
+Description: Block bindings test theme.
+Requires at least: 5.3
+Tested up to: 6.7
+Requires PHP: 5.6
+Version: 1.0
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Text Domain: block-bindings
+Block bindings WordPress Theme, (C) 2021 WordPress.org
```

Comment:
2024 😆

Reply:
Addressed in this commit: https://github.com/WordPress/gutenberg/commit/c7457d0f29f0759993ad85794721ab90896ac351 😄